js/ui/templates/top-blocked-truncated-list-items.es6.js

```diff
@@ -0,0 +1,15 @@
+const bel = require('bel')
+
+module.exports = function (companyListMap) {
+    return companyListMap.map((data) => {
+
+        return bel`<li class="top-blocked__li top-blocked__li--truncated">
+          <div class="top-blocked__pill">
+            <div class="top-blocked__li-site__icon ${data.name.replace('.', '').toLowerCase()}"></div>
```

Comment:
You can get rid of the `data.name.replace()` fn (but keep the `.toLowerCase()`). That was only necessary for one tracker mapping from data/tracker_lists/trackersWithParentCompany.json, it was the entry for `Amazon.com`. That's been switched to just `Amazon` now upstream in the data.